node/resolvers/checkout/index.ts

```diff
@@ -174,6 +188,55 @@ export const queries: Record<string, Resolver> = {
   shipping: (_, args: any, { clients: { checkout } }) => {
     return checkout.simulation(args)
   },
+
+  skuPickupSLAs: async (
+    _: any,
+    { itemId, seller, lat, long, country }: SkuPickupSLAListArgs,
+    ctx: Context
+  ) => {
+    const simulation = await ctx.clients.checkout.simulation({
```

Comment:
does each of these items always exist ("`ctx.clients.checkout.simulation`")?

(I suppose they do, just a sanity check)

Reply:
yes they do